chart/templates/tests/test-api-connection.yaml

```diff
@@ -1,19 +1,28 @@
-{{- if .Values.remoteControlPlane.enabled -}}
 apiVersion: v1
 kind: Pod
 metadata:
   name: "{{ include "job-executor-service.fullname" . }}-test-api-connection"
   labels:
-  {{- include "job-executor-service.labels" . | nindent 4 }}
+    {{- include "job-executor-service.labels" . | nindent 4 }}
   annotations:
     "helm.sh/hook": test
 spec:
+  restartPolicy: Never
```

Comment:
Wondering why we need this (or better: why we haven't needed this before ^^)?

Reply:
It was there before but on a different position, it seems like re ordering the yaml elements prepended it to the top

Reply:
Totally overlooked that, thanks!